src/config/dynamic-theme-fixes.config

```diff
@@ -5007,6 +5029,9 @@ div[role="menu"] div[role="menuitem"][class*=" "] > div > div > div,
 div[role="button"][aria-disabled="true"] > div {
     filter: invert(50%) !important;
 }
+#request-access-icon {
+    filter: invert(93.7%) hue-rotate(180deg) contrast(90.6%) !important
```

Comment:
Why the specific filter here?

Reply:
Because it stands out a lot more with just `INVERT`:
![imagen](https://user-images.githubusercontent.com/71190696/163284332-b83b268c-70c1-4633-8591-c2d0ad3d3619.png)

With this filter, it looks like this:
![imagen](https://user-images.githubusercontent.com/71190696/163284342-32fe55c4-cca8-4c63-9ea9-1a2f57f6f140.png)